src/activities/Elsa.Activities.RabbitMq/Activities/RabbitMqMessageReceived/RabbitMqMessageReceived.cs

```diff
@@ -47,6 +47,21 @@ public class RabbitMqMessageReceived : Activity
             Category = PropertyCategories.Configuration)]
         public string ConnectionString { get; set; } = default!;
 
+
+        [ActivityInput( Order = 3,Category = PropertyCategories.Configuration)]
+        public bool EnableSSL { get; set; }
```

Comment:
Please use `Ssl` for casing acronyms of more than 2 characters within the programming context. When using it for display, of course please do use SSL.